system/include/compat/emmintrin.h

```diff
@@ -1177,6 +1177,18 @@ _mm_set1_epi8(char __b)
   return (__m128i)wasm_i8x16_splat(__b);
 }
 
+static __inline__ __m128i __attribute__((__always_inline__, __nodebug__))
+_mm_setr_epi64(__m64 q0, __m64 q1)
+{
+  return (__m128i){ (long long)q0, (long long)q1 };
+}
+
+static __inline__ __m128i __attribute__((__always_inline__, __nodebug__))
+_mm_setr_epi64(int64_t q0, int64_t q1)
+{
+  return (__m128i)wasm_i64x2_make(q0, q1);
+}
```

Comment:
This looks like it defines the same function twice.. which I guess means the second one will be ignored?   Should we remove one of them?

Also, can we add some kind of test for this?

Reply:
@sbc100 , see https://github.com/emscripten-core/emscripten/pull/22243#issuecomment-2253229600 - in process